src/utils/media-utils.js

```diff
@@ -52,31 +52,40 @@ export const determineMediaDuration = memoizee(url => {
   const div = window.document.createElement('div');
   div.style.display = 'none';
   window.document.body.appendChild(div);
+  let cleanedUp = false;
+  const ensureCleanup = () => {
+    if (!cleanedUp) {
+      ReactDOM.unmountComponentAtNode(div);
+      div.remove();
+      cleanedUp = true;
+    }
+  };
   const playerPromise = new Promise((resolve, reject) => {
     try {
       const element = React.createElement(ReactPlayer, {
         url,
         light: false,
         playing: false,
         onDuration: durationInSeconds => {
-          const durationInMiliseconds = durationInSeconds * 1000;
+          const durationInMiliseconds = Math.ceil(durationInSeconds * 1000);
```

Comment:
I had to do this because for non-Youtube sources we get fractional numbers that tear the antd slider (based on integer milliseconds) into peaces. As I was changing this I also decided to add some cleanup code, and one thing led to each other ...

Reply:
😄 👍